CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Compiler
 
+- Added more detail on the constructor error
+  where a variable starting with an uppercase is defined by let binding.
+  ([Arian Pourarian](https://github.com/Arian94))
```

Comment:
This isn't right. The error message is giving more detail for when a pattern references a value constructor that does not exist.